### PR TITLE
feat: add `get_indexes`

### LINKED ISSUE #10: Fill in `get_indexes`

### PR DESCRIPTION
**Pull request by HuaHuaY**
wait https://github.com/risingwavelabs/risingwave/pull/6636 and https://github.com/risingwavelabs/risingwave/pull/6717 to be merged.

close #10

### CODE REVIEW

File: sqlalchemy_risingwave/base.py [hunk 1]
```diff
@@ -113,7 +113,38 @@ def get_columns(self, conn, table_name, schema=None, **kw):
...
+            conn, table_name, schema, info_cache=kw.get("info_cache")
+        )
+
+        sql = (
+            "select i.relname, a.attname, a.attnotnull from pg_catalog.pg_class t "
+            "join pg_catalog.pg_index ix on t.oid = ix.indrelid "
+            "join pg_catalog.pg_class i on i.oid = ix.indexrelid "
+            "join pg_catalog.pg_attribute a on t.oid = a.attrelid "
+            "where t.oid = :table_oid"
+        )
```
Comment (BowenXiao1999) [inline sqlalchemy_risingwave/base.py:126]: Have you checked how Postgresql implement it? Is our apporach the same with them?

**Review by BowenXiao1999 — APPROVED** [review]
LGTM, Well Done!

But I'm not sure whether Superset will use this `get_indexes` API...

File: sqlalchemy_risingwave/base.py (hunk 1 above)
Comment (HuaHuaY) [reply sqlalchemy_risingwave/base.py:126]: PostgreSQL will add some attributes in the return value dictionary, but our return value is the same as cockroachdb. So sql is simpler than PostgreSQL. But there are some errors in this sql, I missed a condition in "on". I will fix it later.

File: test/test_schema.py [hunk 2]
```diff
@@ -42,9 +29,14 @@ def test_returning_clause(self):
...
+            conn.execute("CREATE TABLE three_columns (id1 INT, id2 INT, id3 INT)")
+            conn.execute("CREATE INDEX three_columns_idx ON three_columns(id2) INCLUDE(id1)")
+
             insp = inspect(testing.db)
-            indexes = insp.get_indexes("users")
+            indexes = insp.get_indexes("three_columns")
+
+            assert len(indexes) == 1
+            assert indexes[0]["name"] == "three_columns_idx"
+            assert indexes[0]["column_names"] == ["id2", "id1"]
```
Comment (BowenXiao1999) [inline test/test_schema.py:40]: Can we get pk of this index?

**Review by BowenXiao1999 — APPROVED** [review]
LGTM

File: test/test_schema.py (hunk 2 above)
Comment (HuaHuaY) [reply test/test_schema.py:40]: Now we can't. We have the ability to add and use `indnkeyatts` column in `pg_index`. We can support getting pk when we need it.

Comment (HuaHuaY) [reply test/test_schema.py:40]: Now we can't. We have the ability to add and use `indnkeyatts` column in `pg_index`. The value of `indnkeyatts` means that how many keys in front of `indkey` are pk. We can support getting pk when we need it.

**Pull request merged**